packages/core/admin/ee/server/controllers/role.js

```diff
@@ -1,14 +1,11 @@
 'use strict';
 
-const { ApplicationError } = require('@strapi/utils').errors;
 const {
   validateRoleCreateInput,
```

Comment:
Are we extending the role controller only because of the custom validation?
If yes, side note for later, but it would be nice to load scoped validators just like we do for services & controller (basically for domain like APIs). It would allow us to require them from a registry (with a dynamic value depending on ee/ce)

Reply:
Yes agreed we can make this simpler